<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1902</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1900</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="imgLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1591</width>
       <height>895</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>1590</x>
       <y>0</y>
       <width>311</width>
       <height>921</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="drawLine">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>93</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>画直线</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearPic">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>870</y>
        <width>93</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QPushButton" name="forward">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>870</y>
        <width>93</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>前进</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backwards">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>870</y>
        <width>93</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>后退</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="lineWidth">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>线宽</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="lineStartXY">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起点XY</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="lineEndXY">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>终点XY</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="lineColor">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>颜色</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1902</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
